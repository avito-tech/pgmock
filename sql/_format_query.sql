select pgmock._format_query('123');

select pgmock._format_query($$"${'schema_name.table_name'::regclass}"$$);

select pgmock._format_query($${"oid": 123}$$);

select pgmock._format_query($${"oid": "${'schema_name.table_name'::regclass}"}$$);
