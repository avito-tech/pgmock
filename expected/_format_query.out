select pgmock._format_query('123');
 _format_query 
---------------
 {"oid": 123}
(1 row)

select pgmock._format_query($$"${'schema_name.table_name'::regclass}"$$);
                  _format_query                   
--------------------------------------------------
 {"oid": "${'schema_name.table_name'::regclass}"}
(1 row)

select pgmock._format_query($${"oid": 123}$$);
 _format_query 
---------------
 {"oid": 123}
(1 row)

select pgmock._format_query($${"oid": "${'schema_name.table_name'::regclass}"}$$);
                  _format_query                   
--------------------------------------------------
 {"oid": "${'schema_name.table_name'::regclass}"}
(1 row)

